README.md

```diff
@@ -8,6 +8,14 @@ libprotobuf-mutator is a library to randomly mutate
 [protobuffers](https://github.com/google/protobuf). <BR>
 It could be used together with guided fuzzing engines, such as [libFuzzer](http://libfuzzer.info).
 
+The core of libprotobuf-mutator has the following dependencies:
+
+- CMake >=3.10
```

Comment:
can we avoid adding another place which can get out of sync
Maybe https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-a-permanent-link-to-a-code-snippet

Reply:
> can we avoid adding another place which can get out of sync?

@vitalybuka it would not be fun to dig out that info, and the damage of these going out of sync is minimal. The reason I'm adding it is because this info is not documented anywhere. I can drop the part on CMake if it takes to much attention, Clang and Protobuf are key here.

> Maybe https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-a-permanent-link-to-a-code-snippet

These links would continue working but also go stale and not gain anything.

Reply:
User will hit error if something is out of date. And then see Cmake condition.
I don't think anyone will start from documentation either before seeing error, or after seeing error.

Reply:
@vitalybuka I'm reading that as a wish to drop CMake versions from this. Dropped and pushed.